cmd/cloak/clientgen.go

```diff
@@ -75,6 +82,19 @@ func ClientGen(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func getLang(cmd *cobra.Command) (string, error) {
+	pkg, err := cmd.Flags().GetString("lang")
+	if err != nil {
+		return "", err
+	}
+
+	// If a package name was provided as a flag, use it
```

Comment:
There should always be a package name provided, since for now the default value of the flag is `Go`

Reply:
True, I will remove the default value once I'm more advanced on the PR. I didn't want to break the CI (magefile calling client-gen without parameters).

Reply:
fixed.